### ISSUE TITLE
warmup_epoch params

### ISSUE THREAD

**Issue by dhm42** [CSTR-Edinburgh/merlin#128]
Can someone please explain to me what is the use of those two parameters and is there is a way to choose their optimal values:
warmup_epoch & warmup_momentum
That we can find in the config files:
[Architecture]
hidden_layer_size: [512, 512, 512, 512]
hidden_layer_type: ['TANH', 'TANH', 'TANH', 'TANH']
#if RNN or sequential training is used, please set sequential_training to True.
sequential_training : False
dropout_rate : 0.0
learning_rate    : 0.002
batch_size       : 256
output_activation: linear
**warmup_epoch     : 10
warmup_momentum  : 0.3**
training_epochs  : 25

Thank you.

**Comment by bajibabu** [comment]
warmup_epoch and warmup_momentum values are used in optimization, they control the learning rate and momentum used in SGD optimization. After warmup epochs the learning rate will decrease by half for every epoch and the momentum value is changed to 0.9. You can see in the code https://github.com/CSTR-Edinburgh/merlin/blob/master/src/run_merlin.py#L264.  You can read more about learning rate decay here https://cs231n.github.io/neural-networks-3/

**Comment by dhm42** [comment]
@bajibabu  Thanks a lot for your explanation !
